services/sonar/sonar-helpers.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const Joi = require('joi')
+const { colorScale } = require('../color-formatters')
+
+const patternBase = ':protocol(http|https)/:host(.+)/:component(.+)'
+const ratingPercentageScaleSteps = [10, 20, 50, 100]
+const ratingScaleColors = [
+  'brightgreen',
+  'yellowgreen',
+  'yellow',
+  'orange',
+  'red',
+]
+const badMetricColorScale = colorScale(
```

Comment:
there's probably better names for these two color scales but these sufficed my needs 😆

Reply:
My suggestion would be `positiveMetric` / `negativeMetric`. Do you think that is any better?

Reply:
yes indeed 👍 will update them accordingly